prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -130,24 +129,25 @@ class FhirPathCustomResolver : FHIRPathEngine.IEvaluationContext, Logging {
         }
 
         // Evaluate the constant before it is used.
-        val value = if (constantValue.isNullOrBlank()) null
+        val values = if (constantValue.isNullOrBlank()) null
         else {
```

Comment:
Add a unit test to test that you can actually get a collection out of this function.

Reply:
To make sure I understand this request, you want to add a test that tests that a third-party library, `FHIRPathEngine`, can return multiple values, even if its signature says it returns a list? In my opinion this test would test a third party library, something I think we shouldn't be doing. I do think we should add another test, per Patricia's comment. In this test, I would mock `FhirPathUtils.evaluate` to return two items: a numeric String and a non-Numeric String and test that our method, `resolveConstant`, handles it correctly. Do you agree? Any additional thoughts?

Reply:
Nope. Don't test the third party library, but you have code here that used to convert one string to a numeric value and not can do multiple. So you need some test to test that. You always assume the third party library works. Here you will need some FHIR Path that will return a collection of string that have numbers on it. Let me know if you need help there.

Reply:
So I just committed a new test called `test fhir path resolver multiple values`. I think that's what you're asking for but please let me know if I'm missing something.